src/main/java/org/broadinstitute/hellbender/utils/read/GoogleGenomicsReadToGATKReadAdapter.java

```diff
@@ -631,6 +633,16 @@ public void clearAttributes() {
 
     @Override
     public GATKRead copy() {
+        // workaround until https://github.com/google/google-http-java-client/issues/293 is released
+        genomicsRead.setAlignedQuality(new ArrayList<>(genomicsRead.getAlignedQuality()));
+        Map<String, List<String>> infoCopy = new HashMap<>();
+        for (Map.Entry<String, List<String>> entry : genomicsRead.getInfo().entrySet()) {
+            infoCopy.put(entry.getKey(), new ArrayList<>(entry.getValue()));
+        }
+        genomicsRead.setInfo(infoCopy);
+        genomicsRead.getAlignment().setCigar(new ArrayList<>(genomicsRead.getAlignment().getCigar()));
+        // end workaround
```

Comment:
In that case, could you please:

-Move the workaround into its own private method `workaroundHttpClientCloneBug()` called from `copy()`, with a comment to delete once the fix is in.

-Alter the workaround so that it won't fail if `genomicsRead.getInfo() == null` or `genomicsRead.getAlignment() == null` or `genomicsRead.getAlignedQuality() == null`